webpush/push.go

```diff
@@ -25,7 +25,7 @@ import (
 
 const (
 	gcmURL     = "https://android.googleapis.com/gcm/send"
-	tempGcmURL = "https://gcm-http.googleapis.com/gcm"
+	tempGcmURL = "https://fcm-http.googleapis.com/fcm/send"
```

Comment:
Would be nice to know what the usage is like of this package. If it's low, then this can probably be removed.